class.jetpack.php

```diff
@@ -3207,10 +3207,14 @@ public static function get_activation_source( $referer_url ) {
 	 * @return null              [description]
 	 */
 	public static function do_version_bump( $version, $old_version ) {
-
 		if ( ! $old_version ) { // For new sites
 			// Setting up jetpack manage
 			Jetpack::activate_manage();
+		} else {
+			// If a Jetpack is still active but not connected when updating verion, remind them to connect with the banner.
+			if ( ! Jetpack::is_active() ) {
+				Jetpack_Options::update_option( 'dismissed_connection_banner', false );
```

Comment:
Maybe we could delete the option altogether instead?
`Jetpack_Options::delete_option( 'dismissed_connection_banner' );`

Reply:
yeah, the only benefit of leaving the presence of an option here would be that we would be able to tell if they connected after being re-shown the banner via this change. Basically being able to measure its success. But we're not doing that, so let's delete it.